tests/Util/yaml_fixtures/complex_string_to_array.test

```diff
@@ -0,0 +1,14 @@
+main:
+    custom_authenticator: App\Security\SomeOtherAuthenticator
+===
+$string = $data['main']['custom_authenticator'];
+$data['main']['custom_authenticator'] = [];
+$data['main']['custom_authenticator'][] = $string;
+$data['main']['custom_authenticator'][] = 'App\Security\AppCustomAuthenticator';
+$data['main']['entry_point'] = 'Entry';
+===
+main:
+    custom_authenticator:
+        - App\Security\SomeOtherAuthenticator
+        - App\Security\AppCustomAuthenticator
+    entry_point: Entry
```

Comment:
Is it possible to add a newline? or is the test failing afterwards?

Reply:
no - ysm test fixtures do not get a new line at the EOF or between the `source`, `changeCode`, and/or the `expected` sections. Makes them go bonkers..